### DISCUSSION TITLE
Admin Control Panel

### DISCUSSION THREAD

**Discussion by GiovanniYuriMita** [hystax/optscale#501] [Q&A] [answered]
Hey OptScale team,

Does OptScale have an Admin Control Panel or something similar?

It would be awesome to have a dashboard where we can view and manage all organizations and users across the entire OptScale deployment. I'm planning to build a unified admin panel using the available APIs, but before diving in, I wanted to check if this feature already exists or is in the works.

Would love to hear your thoughts or any existing solutions!

Thanks!

**Comment by stanfra** [comment, marked as answer]
Hi @GiovanniYuriMita! We don't have this functionality and we don't plan to implement it in the near future, but if you implement it, we will be happy to consider your public request.

**Reply by GiovanniYuriMita** [reply]
Hi @stanfra!

Thank you so much for the quick response and for clarifying my question!

I'll work on this functionality!
